nixos/tests/ceph.nix

```diff
@@ -0,0 +1,147 @@
+{ system ? builtins.currentSystem }:
+
+with import ../lib/testing.nix { inherit system; };
```

Comment:
Can you add yourself to maintainers? https://github.com/NixOS/nixpkgs/blob/master/nixos/tests/mesos.nix#L3

Reply:
I've added myself to the maintainers list, at L145, I should probably restructure the test after that mesos test, I just did it this way because I used networking.nix as a template.